examples/example-script.py

```diff
@@ -59,9 +59,9 @@
     print_("authorized=", client.session.authorized)
     print_("me=", client.users.me())
 
-if 'ASANA_API_KEY' in os.environ:
-    # create a client with your Asana API key
-    client = asana.Client.basic_auth(os.environ['ASANA_API_KEY'])
+if 'ASANA_PERSONAL_ACCESS_TOKEN' in os.environ:
```

Comment:
eeh, maybe a little long? Perhaps just `ASANA_ACCESS_TOKEN`? You shortened the client method above; if it's appropriate there, seems so here too.